dpnp/backend/extensions/common/ext/details/common_internal.hpp

```diff
@@ -23,15 +23,15 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 //*****************************************************************************
 
-#include "common.hpp"
+#include "ext/common.hpp"
 #include "utils/type_dispatch.hpp"
 #include <pybind11/pybind11.h>
 
 namespace dpctl_td_ns = dpctl::tensor::type_dispatch;
 
-namespace statistics::common
+namespace ext::common
 {
-size_t get_max_local_size(const sycl::device &device)
+inline size_t get_max_local_size(const sycl::device &device)
```

Comment:
Do we need to remove declaration in `dpnp/backend/extensions/common/ext/common.hpp` then?
The same is applicable for below updated functions.

Reply:
No. We still need it as forward declaration.

Reply:
Oh, I see